ompi/mpi/man/man3/MPI_Win_create_dynamic.3in

```diff
@@ -53,25 +53,28 @@ Fortran only: Error status (integer).
 .SH DESCRIPTION
 .ft R
 MPI_Win_create_dynamic is a one-sided MPI communication collective call executed by all processes in the group of \fIcomm\fP. It returns a window object without memory attached that can be used by these processes to perform RMA operations.
+.sp
+A window created with \fBMPI_Win_create_dynamic\fP requires the \fItarget_disp\fP argument for all RMA communication functions to be the actual address at the target.
+
 .sp
 The following info keys are supported:
 .ft R
 .TP 1i
 no_locks
 If set to \fItrue\fP, then the implementation may assume that the local
 window is never locked (by a call to MPI_Win_lock or
-MPI_Win_lock_all).  Setting this value if only active synchronization
+MPI_Win_lock_all). Setting this value if only active synchronization
 may allow the implementation to enable certain optimizations.
 .sp
 .TP 1i
 accumulate_ordering
 By default, accumulate operations from one initiator to one target on
-the same window are strictly ordered.  If the info key
+the same window are strictly ordered. If the info key
```

Comment:
I just spotted this: The statement 

> By default, accumulate operations from one initiator to one target on
the same window are strictly ordered

is incorrect. It should be:

> By default, accumulate operations from one initiator to one target on
the same window memory location are strictly ordered.

@zerothi I know this didn't come out of your PR but would you mind fixing the language while you're at it? (probably applies to `MPI_Win_allocate` and `MPI_Win_create` as well)

Reply:
done (wasn't in win_allocate)